Fix:

<inline asm>:1:1: error: unknown directive
.symver logf,logf@GLIBC_2.2.5
^
<inline asm>:2:1: error: unknown directive
.symver powf,powf@GLIBC_2.2.5
^
<inline asm>:3:1: error: unknown directive
.symver memcpy,memcpy@GLIBC_2.2.5
^
<inline asm>:4:1: error: unknown directive
.symver memmove,memmove@GLIBC_2.2.5
^

This problem has been reported to the developer by email.
--- src/xwin.c.orig	2020-06-19 07:05:40.000000000 -0500
+++ src/xwin.c	2020-06-26 11:56:48.000000000 -0500
@@ -1,16 +1,3 @@
-#if __LP64__
-# define SYMVER "GLIBC_2.2.5"
-#else
-# define SYMVER "GLIBC_2.0"
-#endif
-#define USE_OLD_SYM(F,V) __asm__(".symver " #F "," #F "@" V)
-USE_OLD_SYM(logf,SYMVER);
-USE_OLD_SYM(powf,SYMVER);
-USE_OLD_SYM(memcpy,SYMVER);
-USE_OLD_SYM(memmove,SYMVER);
-
-//__asm__(".symver memcpy,memcpy@GLIBC_2.2.5");
-//__asm__(".symver memmove,memmove@GLIBC_2.2.5");
 #ifdef VMS
 #include "decw$include:xlib.h"
 #include "decw$include:xutil.h"
